cypress/e2e/patient_spec/encouter-notes.cy.ts

```diff
@@ -0,0 +1,78 @@
+import { FacilityCreation } from "@/pageObject/facility/FacilityCreation";
+import { PatientEncounter } from "@/pageObject/Patients/PatientEncounter";
+
+const facilityCreation = new FacilityCreation();
+const patientEncounter = new PatientEncounter();
+
+describe("Encounter Notes - Threads, Messages, and Notes", () => {
+  beforeEach(() => {
+    cy.loginByApi("devdoctor"); // logging in as doctor
```

Comment:
```suggestion
    cy.loginByApi("devdoctor"); // logging in as doctor
```

- since we are connected to develop backend temporarily, create a new user manually in the develop, add it to the fixture file and use that in your test